### PR TITLE
Install rubocop-rspec

### PR DESCRIPTION
**Pull request by mwolman**
### Description:
* Install `rubocop-rspec` gem
* Fix some offenses with the autocorrect option and others manually
* Add the pending ones to the `.rubocop_todo.yml` file. These offenses are going to take more time and should be done in others pull requests

### CODE REVIEW

**Review by diebas — APPROVED** [review]
loving it!

File: .rubocop.yml [hunk 1]
```diff
@@ -65,6 +68,7 @@ Style/Documentation:
 Style/FrozenStringLiteralComment:
   Enabled: false
 
+
```
Comment (lucaleivaloop) [inline .rubocop.yml:71]: Mind checking this file? It seems like it has a lot of extra blank lines.

Comment (mwolman) [reply .rubocop.yml:71]: I added 2 empty lines on purpose, to identify easier the 3 sections: ruby, rails and rspec 👀 🙈

Comment (lucaleivaloop) [reply .rubocop.yml:71]: Can't each gem read its own file?

Comment (mwolman) [reply .rubocop.yml:71]: I think that there's a way to do that 🤔  I'll try and let you know what happens

**Review by lucaleivaloop — APPROVED** [review]

**Review by mconiglio — APPROVED** [review]

**Review by aperezcristina — APPROVED** [review]

**Pull request merged**